wikimedia/wikipedia-ios
# Prevent iPad Scribble on Explore feed search bar from crashing app

**Phabricator:** https://phabricator.wikimedia.org/T269480

### Notes
A Pencil Scribble on the Explore feed search bar was causing a crash on supported devices (iOS 14 iPad's that support Scribble via Pencil). This seemed to be occurring because our Search bar in the Explore feed is mostly an ornamental placeholder – on tap, a new text field and view is displayed. This appeared to be the root of the Pencil Scribble crash - the Explore feed search bar starts accepting the scribble as first responder, but because it disappears immediately the Scribble seems to get tripped up. This appeared to be causing the Pencil Scribble crash. I tried to figure out how we could maybe pass the user's Scribble interaction into the new text field, but the Pencil stuff seems too opaque for me to figure out this kind of behavior.

So as of currently with this fix, finger based behavior is unchanged (no difference when tapping the field on iPad or iPhone to expose the actual search text field and type with software keyboard) and Scribble still works, but requires an initial tap via Pencil or finger of the Explore feed search bar field before being able to Scribble in the actual search field. 

### Test Steps
To test with Apple Pencil or stylus that supports Scribble:

1. Open app, navigate to Explore feed.
2. Try to scribble on the Explore feed's top search bar field, it shouldn't allow you to.
3. Tap the top search bar to make it focused – a transition animation occurs. Now try and scribble on the field. It should allow you to and not crash.

Without a Pencil, regression testing can be performed just using your finger to tap the Explore feed search bar field to ensure it still behaves as it did previously.
